### ISSUE TITLE
deepcopy of `pyasn1.type.useful.FixedOffset` instance raises exception

### ISSUE THREAD

**Issue by tadams42** [etingof/pyasn1#81]
Hi, here is how to reproduce it:

~~~python
from copy import deepcopy

import pyasn1
from pyasn1.type.useful import GeneralizedTime

pyasn1.__version__
# '0.3.6'

dt = GeneralizedTime("20170916234254+0130").asDateTime
deepcopy(dt)
~~~

rises `TypeError: __init__() takes exactly 3 arguments (1 given)` in Python 2.7 or `TypeError: __init__() missing 2 required positional arguments: 'offset' and 'name'` in Python 3.5

**Comment by etingof** [comment]
Thank you for your report!

I am being able to reproduce it. Fix is coming. ;)

**Comment by etingof** [comment]
Fixed in master (03201a7722ef13cc4f051f0d67aac76eecc30aa5). Apparently, that's the tzinfo class quirk.
